src/components/Card/Card.tsx

```diff
@@ -66,6 +66,7 @@ export const Card = ({
         active
         loading={isLoading}
         paragraph={Card.skeletonParagraph}
+        rootClassName={skeleton}
       >
         {(title || subtitle) && <div className={header}>
```

Comment:
So we are not showing the card title while loading?

Reply:
yes, by design